dexbot/strategies/external_feeds/ccxt_feed.py

```diff
@@ -1,123 +1,106 @@
-    await exchange.load_markets()
-    print(exchange.id, 'fetching all tickers by simultaneous multiple concurrent requests')
-    symbols_to_load = get_active_symbols(exchange)
-    input_coroutines = [fetch_ticker(exchange, symbol) for symbol in symbols_to_load]
-    tickers = await asyncio.gather(*input_coroutines, return_exceptions=True)
-    for ticker, symbol in zip(tickers, symbols_to_load):
-        if not isinstance(ticker, dict):
-            print(exchange.id, symbol, 'error')
-        else:
-            print(exchange.id, symbol, 'ok')
-    print(exchange.id, 'fetched', len(list(tickers)), 'tickers')
-
-
 async def print_ticker(symbol, id):
     # verbose mode will show the order of execution to verify concurrency
     exchange = getattr(accxt, id)({'verbose': True})
     print(await exchange.fetch_ticker(symbol))
+    await exchange.close()
 
 
-###### unit tests ######
+# unit tests
 @click.group()
 def main():
     pass
 
 
 @main.command()
-def test_async():
-
+def test_async2():
+    """
+    get all tickers from multiple exchanges using async
+    """
     symbol = 'ETH/BTC'
     print_ethbtc_ticker = functools.partial(print_ticker, symbol)
     [asyncio.ensure_future(print_ethbtc_ticker(id)) for id in [
-            'bitfinex',
-            'binance',
-            'kraken',
-            'gdax',
-            'bittrex',
-            ]]
+        'bitfinex',
+        'binance',
+        'kraken',
+        'gdax',
+        'bittrex',
+    ]]
     pending = asyncio.Task.all_tasks()
     loop = asyncio.get_event_loop()
     loop.run_until_complete(asyncio.gather(*pending))
 
-#    asyncio.get_event_loop().run_until_complete(fetch_tickers(accxt.bitfinex({
-#                    'enableRateLimit': True,  # this option enables the built-in rate limiter
-#                    })))
 
+@main.command()
+def test_async():
+    """
+    get ticker for bitfinex using async
+    """
+    bitfinex = accxt.bitfinex({'enableRateLimit': True, })
+    ticker = asyncio.get_event_loop().run_until_complete(
+        fetch_ticker(bitfinex, 'BTC/USDT'))
+    print(ticker)
 
 
 @main.command()
 @click.argument('exchange')
 @click.argument('symbol')
 def test_feed(exchange, symbol):
-    '''
-    Usage: exchange [symbol]   
+    """
+    Usage: exchange [symbol]
     Symbol is required, for example:
     python ccxt_feed.py test_feed gdax BTC/USD
-    ''' 
-    usage = "Usage: python ccxt_feed.py id [symbol]\nSymbol is required, for example: python ccxt_feed.py gdax BTC/USD"
-
+    """
+    usage = "Usage: python ccxt_feed.py id [symbol]"
     try:
         id = exchange  # get exchange id from command line arguments
-
-        # check if the exchange is supported by ccxt
         exchange_found = id in ccxt.exchanges
-
+        # check if the exchange is supported by ccxt
         if exchange_found:
             print_args('Instantiating', green(id))
-
             # instantiate the exchange by id
             exch = getattr(ccxt, id)()
```

Comment:
`exch` -> `exchange` and ccxt, could be named better if possible. If not, write a comment that tells better what ccxt is.